### ISSUE TITLE
Ability to change `snappable` or `snapThreshold` value after initializing moveable

### ISSUE THREAD

**Issue by seahindeniz** [daybrush/moveable#553]
## Environments
* Moveable Component version: 0.27.1

## Description
There are some cases when I want to be able to change `snappable` or `snapThreshold` while dragging an element.
I have tried following way, but that didn't work:
```js
const moveableOptions = {
  // ...
  snappable: true,
  snapThreshold: 5,
};
const moveable = new Moveable(container, moveableOptions);
const snapCount = 0
const snapHandler = () => {
  if (snapCount !== 100) return;

  moveableOptions.snapThreshold = 0;
  moveable.snapThreshold = 0;

  moveable.updateTarget();
};

this.moveable.on('snap', snapHandler);
```

Or

```js
const moveableOptions = {
  // ...
  snappable: true,
  snapThreshold: 5,
};
const moveable = new Moveable(container, moveableOptions);
const snapCount = 0
const snapHandler = () => {
  if (snapCount !== 100) return;

  moveableOptions.snappable = false;
  moveable.snappable = false;

  moveable.updateTarget();
};

this.moveable.on('snap', snapHandler);
```

**Comment by daybrush** [comment]
@seahindeniz 

What doesn't change is a bug. The dependency seems to be the problem. I'll check.

**Comment by daybrush** [comment]
@seahindeniz 


* `moveable` 0.27.2
* `react-moveable` 0.30.2
* `preact-moveable` 0.29.2
* `ngx-moveable` 0.23.2
* `svelte-moveable` 0.18.2
* `lit-moveable` 0.4.2
* `vue-moveable` 2.0.0-beta.5
* `vue3-moveable` 0.2.2


moveable's new version is released. Check it again.

**Comment by seahindeniz** [comment]
Thanks, @daybrush, I just tried it works as expected! 🎉 ✨ 

However, after I tried to print/log its status, I have realized that the change doesn't happen instantly/synchronously, I think.
To get the state updated from `snappable` property, the flow needs to be waited like ˜0.0015 milliseconds. I know, it's not a big deal, but it's there, just to let you know.
Here https://codesandbox.io/s/brave-sun-vvptl?file=/src/index.ts
The outputs in the console, shows the previous value, value after state gets changed and the changed value after waiting for some milliseconds.


Iterating following lines with 5 seconds interval:
```js
console.log(moveable.snappable);
moveable.snappable = !moveable.snappable;
console.log(moveable.snappable);
await delay(10);
console.log(moveable.snappable);
```
Outputs:
```bash
00:29:00.004 true
00:29:00.006 true
00:29:00.019 false

00:29:05.007 false
00:29:05.008 false
00:29:05.019 true

00:29:10.003 true
00:29:10.004 true
00:29:10.014 false

00:29:15.003 false
00:29:15.004 false
00:29:15.015 true
```
